### ISSUE TITLE
LiveSync - XML changes cause restart on device and does not update - Android P beta possibly

### ISSUE THREAD

**Issue by bradmartin** [NativeScript/NativeScript#6046]
### Did you verify this is a real problem by searching the [NativeScript Forum](http://forum.nativescript.org) and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?

Yes

### Tell us about the problem
Please, ensure your title is less than 63 characters long and starts with a capital
letter.

### Which platform(s) does your issue occur on?

Android

### Please provide the following version numbers that your issue occurs with:
- CLI: 4.1.2
- Cross-platform modules: 4.1.0
- Runtime(s): 4.1.3
- Plugin(s): none

### Please tell us how to recreate the issue in as much detail as possible.

Create a new test app (you can't use playground, unable to reproduce it there because it seems livesync related). Run the app on a Pixel2 running the Android P beta. I cannot reproduce this on an emulator. 

### Is there code involved? No.

I originally thought the issue related to Google app backup (discovered in May 2017 https://github.com/NativeScript/NativeScript/issues/4137#issuecomment-304710742) was the issue with a recent update to Android P beta. However, I've disabled app backup and put the phone in airplane mode and livesync still restarts the app with XML changes and the changes are not reflected in the running app.

I have tried 3 emulators, none are running the Android P beta and everything works fine (no restart and is successful to sync changes) so the only thing I can narrow down right now is a device running the Android P beta. This wasn't an issue a month or two ago as I've been using the Android P beta since it was opened up for download. There was a recent beta update (last week I believe, possibly July 5th 2018) and this might be the one that introduced the issue.

To summarize:

- Create new NS app
- Run on device with Android P beta (latest beta version)
- change XML content
- livesync will restart the app (should just sync without restarting)
- Changes are _not_ reflected after restart.

**Comment by bradmartin** [comment]
If I bump the version code in the app manifest forcing an android build, then the changes will sync and reflect but of course this defeats the livesync purpose 😄

**Comment by tsonevn** [comment]
Hi, @bradmartin,
This issue seems to be duplicated of [#3602](https://github.com/NativeScript/nativescript-cli/issues/3602)

**Comment by bradmartin** [comment]
Ahh, my bad for not seeing that issue. Thanks Nikolay :)
